core/src/main/java/org/apache/iceberg/CachingCatalog.java

```diff
@@ -70,6 +70,31 @@ public Table loadTable(TableIdentifier ident) {
     return tableCache.get(canonicalizeIdentifier(ident), catalog::loadTable);
   }
 
+  @Override
+  public boolean createNamespace(Namespace namespace) {
+    return false;
```

Comment:
These should delegate to the wrapped catalog.

Reply:
thanks Ryan , now has fixed.